awx/main/models/ha.py

```diff
@@ -145,7 +146,12 @@ def get_absolute_url(self, request=None):
 
     @property
     def consumed_capacity(self):
-        return sum(x.task_impact for x in UnifiedJob.objects.filter(execution_node=self.hostname, status__in=('running', 'waiting')))
+        capacity_consumed = sum(
+            x.task_impact
+            for x in UnifiedJob.objects.filter((Q(controller_node=self.hostname) | Q(execution_node=self.hostname)) & Q(status__in=('running', 'waiting')))
```

Comment:
Do you intend the filter here to return duplicates? Because if a job is on a hybrid node, it _should_ be double-counted. It should be counted by `task_impact` for the execution part, and the constant value for the control part, so at first glance I don't see how this is going to work.

What is the intended capacity consumption for the control nodes? I'd like to start with some docs about that.

Reply:
I have addressed this on hybrid nodes in https://github.com/ansible/awx/pull/11651/commits/1fe442270db44279fe3c64cbcdcbd1e20b959709